### PR TITLE
NXBT-2589: conditionally add azure symlink

### PR DESCRIPTION
**Pull request by mguillaume**

### CODE REVIEW

**Comment by nelsonsilva** [comment]
👍

File: roles/slave_tools/tasks/main.yml [hunk 1]
```diff
@@ -274,6 +274,14 @@
   npm: name=azure-cli version=0.10.7 global=yes
   register: azure_install
   tags: azure-cli
+- name: Check for azure binary
+  stat: path=/usr/local/bin/azure
+  register: azureinbin
+  tags: azure-cli
+- name: Add azure symlink
+  file: path=/usr/local/bin/azure src=/opt/build/tools/node-v10.13.0-linux-x64/bin/azure state=link
```
Comment (jcarsique) [inline roles/slave_tools/tasks/main.yml]: could we get this path from nodejs install?
https://github.com/nuxeo/nuxeo-ansible-jenkins/blob/25210953f4e2f46fe22ae732aa2cce18f82e3ed7/roles/slave_tools/tasks/main.yml#L235

**Comment by jcarsique** [comment]
:+1: 
see comment about nodejs path

File: roles/slave_tools/tasks/main.yml (hunk 1 above)
Comment (mguillaume) [reply roles/slave_tools/tasks/main.yml]: Eventually, but that will probably happen post ansible upgrade, when we put in place the system for "custom" slaves.

**Pull request merged**